conviva/src/main/java/com/bitmovin/analytics/conviva/ConvivaAnalyticsIntegration.java

```diff
@@ -649,30 +516,15 @@ public void onEvent(PlayerEvent.TimeShifted timeShiftedEvent) {
     };
 
     private void setSeekStart(int seekTarget) {
-        if (!isSessionActive()) {
-            // Handle the case that the User seeks on the UI before play was triggered.
-            // This also handles startTime feature. The same applies for onTimeShift.
-            return;
-        }
         Log.d(TAG, "Sending seek start event");
-        try {
-            playerStateManager.setPlayerSeekStart(seekTarget);
-        } catch (ConvivaException e) {
-            Log.e(TAG, e.getLocalizedMessage());
-        }
+        // TODO - Verify...Conviva docs indicate SEEK_START rather than SEEK_STARTED...?
+        convivaVideoAnalytics.reportPlaybackMetric(ConvivaSdkConstants.PLAYBACK.SEEK_STARTED, seekTarget);
     };
 
     public void setSeekEnd() {
-        if (!isSessionActive()) {
-            // See comment in setSeekStart
-            return;
-        }
         Log.d(TAG, "Sending seek end event");
-        try {
-            playerStateManager.setPlayerSeekEnd();
-        } catch (ConvivaException e) {
-            Log.e(TAG, e.getLocalizedMessage());
-        }
+        // TODO - Verify...Conviva docs indicate SEEK_END rather than SEEK_ENDED...?
```

Comment:
As far as I can see, `SEEK_END` doesn't exist in `ConvivaSdkConstants`, so should just be an error in the documentation, right?